Maintain siblings order on CSV import, refs #13414

jraddaoui opened the pull request:
Maintain siblings order when the nested set update is disabled.
Add a placeholder lft value incrementing the max lft value from
the resource's siblings. This placeholder will overlap with the
parent's rgt value until the nested set is rebuilt.

sbreker commented on lib/model/QubitInformationObject.php:
```diff
@@ -225,6 +225,50 @@ public function save($connection = null)
+      {
+        $connection->rollBack();
+
+        throw $e;
```
What state will io be in if this rollback and exception are thrown? parent::save() is called before, but all child records etc are called after. Will the io be saved, but all child recs not be created?

jraddaoui replied:
@sbreker, that's a great point, thanks!

I'll move down in the method, just before indexing.

sbreker reviewed (commented):
Added one question regarding save() behaviour if the exception is thrown

jraddaoui committed:
Maintain siblings order on CSV import, refs #13414

Maintain siblings order when the nested set update is disabled.
Add a placeholder lft value incrementing the max lft value from
the resource's siblings. This placeholder will overlap with the
parent's rgt value until the nested set is rebuilt.

sbreker reviewed (approved):
Thanks for making that change @jraddaoui !  Looks good
